src/main/java/org/jmeterplugins/repository/JARSourceHTTP.java

```diff
@@ -109,6 +133,11 @@ private AbstractHttpClient getHTTPClient() {
     }
 
     protected JSON getJSON(String uri) throws IOException {
+        PluginsRepo repo = getRepoCache(uri);
+        if (repo != null && repo.isActual()) {
+            log.info("Found cached repo");
+            return JSONSerializer.toJSON(repo.getRepoJSON(), new JsonConfig());
+        }
 
         log.debug("Requesting " + uri);
```

Comment:
Can we make this log and log above INFO level?  It'll help for check that this cache is work

Reply:
Yes, we can